CMakeLists.txt

```diff
@@ -1,16 +1,19 @@
-# Copyright 2017 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+# Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
 #
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+# This file is part of eProsima Fast DDS Shapes-Demo.
```

Comment:
```suggestion
# This file is part of eProsima Fast DDS Shapes Demo.
```

Seeing the README and the documentation, it seems to be written without the hyphen. Please, apply to all this PR.

Reply:
attempt in commit 144eebb